src/types.ts

```diff
@@ -50,9 +50,14 @@ export interface CycliRecipe {
 
 export type Platform = 'android' | 'ios'
 
+export type Mode = 'debug' | 'release'
```

Comment:
Shouldn't it be in build recipe file?

Reply:
moved in ef4d52054da13215e5dbe33ba57e407e5fe26939